deps/luajit.cmake

```diff
@@ -255,9 +255,25 @@ if(MINILUA_USE_LIBM)
   TARGET_LINK_LIBRARIES(minilua m)
 endif()
 
+set(GIT_FORMAT %ct)
+if (CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
+  set(GIT_FORMAT %%ct)
+endif()
+add_custom_command(OUTPUT ${LUAJIT_DIR}/src/luajit_relver.txt
+  COMMAND git show -s --format=${GIT_FORMAT} > ${LUAJIT_DIR}/src/luajit_relver.txt
+  WORKING_DIRECTORY ${LUAJIT_DIR}
+)
+
+add_custom_command(OUTPUT ${LUAJIT_DIR}/src/luajit.h
+  COMMAND ${CMAKE_BINARY_DIR}/minilua host/genversion.lua
```

Comment:
I think this can just be
```cmake
COMMAND minilua host/genversion.lua
```

That's what the command to generate `buildvm_arch.h` does below.

Reply:
OK, Let me have a try.